### DISCUSSION TITLE
Large print file size

### DISCUSSION THREAD

**Discussion by eflehder1** [leap71/PicoGK#52] [Coding for (Computational) Engineers]
Hi @leap71 ,

In order to transfer a print file to a manufacturer, I have tried converting my geometry to an STL or a CLI file. In both cases the size ends up at 24 GB using the print resolution. Is there an approach to reduce this file size, atleast when only using implicit geometry only? 

Best,
Eric

**Comment by LinKayser** [comment]
Not really. But a CLI file should not be an issue for the manufacturer, as you only deal with one slice at a time. There are non-standard ways to transfer implicit-only geometry, but I am not aware of anything that is open. But the main problem with these approaches is that they assume that everything comes from just a long list of implict formulas stacked on top of each other. The moment you break that chain, by doing something non-implicit, the files would become similar in size.

But we are happy to help sort it out with the manufacturer, if you like. CLI files should not pose an issue for them, even if the size of it is large.

Lin
